aider/website/docs/usage/tips.md

```diff
@@ -25,6 +25,10 @@ but be selective.
 Do them one at a time. 
 Adjust the files added to the chat as you go: `/drop` files that don't need any more changes, `/add` files that need changes for the next step.
 
+## Creating new files
+
+If you want aider to create a new file, add it to the repository first with `/add <file>`, so aider knows this file exists and will write to it. Without this, aider might write the changes to an existing files, even when explicitly asked not to do this.
```

Comment:
Just two sentences and I still manage to put an error in them: `existing files` -> `existing file` I'll send a spelling error PR in a moment.